challenges/primes.js

```diff
@@ -10,3 +10,32 @@
 */
 
 // YOUR CODE HERE
+
+let isPrime = function(num) {
+	var factors = 0; 
+	for (let i=2; i<num; i++){
+		if (num%i==0){
+			factors+=1; 
```

Comment:
You keep track of this factors variable, but to make the code run faster, as soon as you find a single number that is a factor of `num`, you could return false.